taxi/views.py

```diff
@@ -1,3 +1,15 @@
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
+from .models import Driver, Manufacturer, Car
```

Comment:
When importing more than two items, it's advisable to write each on a separate line. Additionally, imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability.

Example:

From taxi.models import ( 
	Driver,
	Manufacturer,
	Car
)
Read more about topic [here](https://peps.python.org/pep-0008/#imports)